cubeb-backend/tests/test_capi.rs

```diff
@@ -205,16 +195,18 @@ fn test_ops_context_enumerate_devices() {
         unsafe { OPS.enumerate_devices.unwrap()(c, 0, &mut coll) },
         ffi::CUBEB_OK
     );
-    assert_eq!(coll.device, 0xDEAD_BEEF as *mut _);
-    assert_eq!(coll.count, usize::max_value())
+    assert_ne!(coll.device, std::ptr::null_mut());
+    assert_eq!(coll.count, 1)
 }
 
 #[test]
 fn test_ops_context_device_collection_destroy() {
     let c: *mut ffi::cubeb = get_ctx();
+    let mut device_infos = ManuallyDrop::new(Box::new([DeviceInfo::default().into()]));
```

Comment:
I guess the code is simpler with `ManuallyDrop`, but I wonder if the ownership transfer would be slightly clearer with`Box::into_raw`.